{"uuid": "981e7e5f-956b-4dcc-8768-7f6d6198a1ee", "children": ["6e63e270-00f4-4062-9a79-73f3eac0995f"], "befores": [{"name": "update_food_init", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range\n", "trace": "  File \"c:\\tools\\python 36\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"c:\\tools\\python 36\\lib\\site-packages\\_pytest\\fixtures.py\", line 1122, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"c:\\tools\\python 36\\lib\\site-packages\\_pytest\\fixtures.py\", line 924, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"C:\\DeliverySystem完整代码\\testCase\\conftest.py\", line 72, in update_food_init\n    idMenu = food_init.query({\"page\": 1, \"limit\": 1})['data']['records'][0]['category_id']\n"}, "start": 1705741679232, "stop": 1705741679362}], "start": 1705741679232, "stop": 1705741679369}